frontend/src/app/main/data/plugins.cljs

```diff
@@ -0,0 +1,42 @@
+;; This Source Code Form is subject to the terms of the Mozilla Public
+;; License, v. 2.0. If a copy of the MPL was not distributed with this
+;; file, You can obtain one at http://mozilla.org/MPL/2.0/.
+;;
+;; Copyright (c) KALEIDOS INC
+
+(ns app.main.data.plugins
+  (:require
+   [app.plugins.register :as pr]
+   [potok.v2.core :as ptk]))
+
+(defn open-plugin!
+  [{:keys [plugin-id name description host code icon permissions]}]
+  (try
+    (.ɵloadPlugin
+     js/window
```

Comment:
should be using app.util.global instead js/window

Reply:
ok